### ISSUE TITLE
Warning for `mergeable-selectors` for animations in separate keyframe at-rules.

### ISSUE THREAD

**Issue by dustindowell22** [sasstools/sass-lint#559]
```scss
@keyframes pop {
  0%   { transform: scale(0); opacity: 0; }  // getting warning to merge this with the 0% below
  50%  { transform: scale(1.05); }
  75%  { transform: scale(0.95); }
  100% { transform: scale(1); opacity: 1; }  // getting warning to merge this with the 100% below
}

@keyframes active-left {
  0% { transform: translateX(-100%) rotate(45deg); } // getting warning to merge this with the 0% above
  100% { transform: translateX(0%) rotate(45deg); }  // getting warning to merge this with the 100% above
}
```

`mergeable-selectors` in animations should be scoped to the at-rule.

**Comment by DanPurdy** [comment]
Hi there @dustindowell22 

I've submitted a PR today #556 to fix the no-mergeable-selectors for the new version of Gonzales which we should be releasing soon and I can confirm that keyframes are no longer affected by the rule. :+1: 

So keep an eye out for the 1.6 update :smile:

**Comment by dustindowell22** [comment]
Awesome! Perfect timing. Thanks!

**Comment by DanPurdy** [comment]
@dustindowell22 Sass-lint 1.6 fixes this issue 👍 thanks for your patience!
